graph/graph_joint_inputs.py

```diff
@@ -0,0 +1,61 @@
+import pandas as pd
+import matplotlib.pyplot as plt
+import numpy as np
+import sys
+import os
+import json
+
+def plot_joint_positions(time_of_traj_gen, traj_gen_joint_information, motion_time, motions_positions, joint_idx, filename):
+    plt.figure(figsize=(20, 16))
+    plt.plot(motion_time, motions_positions, label='motion plan', linestyle='--')
+    plt.plot(time_of_traj_gen, traj_gen_joint_information, label='traj gen', linewidth=2)
+    plt.title(f'Joint {joint_idx} Position over Time')
+    plt.xlabel('Time (s)')
+    plt.ylabel('Position (rad)')
+    plt.legend()
+    plt.grid(True)
+    plt.tight_layout()
+    plt.savefig(filename)
+    plt.close()
+
+config_path = "config.json"
```

Comment:
I'm not sure what this is or why an extra config would be necessary.  You are already in the module this script should be able to know where the paths are

Reply:
so suppose we running in root for some reason (this is incorrect but could happen) in this case `~/.viam/...` is not even correct.

Reply:
you are right that this module itself is responsible for creating the files we wish to consume so perhaps it should be possible to know what the paths are, but we still need to specify the unix timestamp at the very least as well as the part-id.